modules/text_server_adv/text_server_adv.cpp

```diff
@@ -2558,7 +2558,9 @@ int64_t TextServerAdvanced::_font_get_spacing(const RID &p_font_rid, SpacingType
 		return fdv->extra_spacing[p_spacing];
 	} else {
 		FontAdvanced *fd = font_owner.get_or_null(p_font_rid);
-		ERR_FAIL_NULL_V(fd, 0);
+		if (!fd) {
```

Comment:
Those Changes are made to avoid 40 spam errors while changing the project font from ProjectSettings.